.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "airbnb-base",
```

Comment:
According to the [contribution guidelines](https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/wiki/Contributing), the extended preset should be [Standard](https://standardjs.com/).

Reply:
Huh, fair enough, I went off the existing eslintrc.js file.

I will say that it seems a bit counter intuitive to have an open issue about moving from `var` to `let` and `const`, but using a linter ruleset that allows using `var`.

If that is part of the aim for updating the codebase it might make sense to change to a more appropriate linting setup in my opinion?

Reply:
These are things rcgy will take a look at and hash out in the morning, I definitely agree a linting system that throws alerts for vars would be good

Reply:
Standard versus AirBnb is indeed a tricky question. I've been using Standard as it was the simplest, and I didn't need to worry about other people's thoughts on the matter. Now that it's not just me developing it, though, I feel like it's perhaps time to shift to a more widely accepted style guide.

Reply:
Both Standard and airbnb are popular styles. If most of the codebase is already using Standard, I'd say that's the one to go with.

Reply:
We can also set up a git hook that automatically fixes all autofixable lint issues on commit, and rejects the commit if any unfixable issues are found.

Reply:
Yeah, the codebase is using Standard at the moment. I'm aware that IAS is not perfect, and it's far from futureproof, and that's one of the reasons that I'm considering AirBnb. Really, my main concern is that Standard alienates the project from people interested in contributing.

Reply:
As Glinkis says, Standard is popular and is used by some big repos. I don't think it'll necessarily scare anyone away who is serious about wanting to contribute.

Really it's a decision to be made on what style you want to enforce on the project.  
Should probably have had more of a discussion in the issue before I made the PR on this, but there's no rush to merge anything in.

Reply:
Well, if that's the case, I reckon that we'll stick with Standard, just to avoid any confusion.

Reply:
Done!